templates/root/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <title>Community Solid Server</title>
+  <link rel="stylesheet" href="/.well_known/css/styles/main.css" type="text/css">
+</head>
+<body>
+  <header>
+    <img src="/.well_known/css/images/solid.svg" alt="[Solid logo]" />
+    <h1>Community Solid Server</h1>
+  </header>
+  <main>
+    <h1>Welcome to the Community Solid Server</h1>
+    <p>
+      This server implements
+      the <a href="https://solid.github.io/specification/protocol">Solid protocol</a>,
+      so you can set your own Solid pods.
+    </p>
+    <p>
+      Some things you can try:
+    </p>
+    <ul>
+      <li>
+        Read or write documents in the <strong>default pod</strong>.
+        <ul>
+          <li>This is the pod you are reading from now.</li>
+          <li>Anyone can read or write;
+              update the <a href="/.acl">access controls</a> to change this.</li>
+          <li>You can disable the default pod in the configuration.</li>
```

Comment:
There is no way to disable the default pod in the configuration. The only way to disable it is to write an acl file to it that prevents all access.

Reply:
We can reconfigure the whole thing though, right? Like map the user pod to the root?

Reply:
Not really. Depends a bit on how you interpret it.

The root container is always going to exist (and has to since it's the root folder if we use a file system). You can use that container as your personal pod. But that requires you to manually put the correct resources there to have a valid acl and webid. You would also have to manually register with the IDP. With manually I mean the registration solution where you manually add an identifying triple to your webid for the IDP. There is no way for the IDP to "generate a pod" in the root container. If you generate a pod with the IDP registration form it is always going to be in a subcontainer (either alice.test.com or test.com/alice depending on your config).

The one thing you can do is disable the IDP so no other pods can be made, but then you also can't use your own server as IDP for your own pod since those are tied together. Might be an idea for the future to have a configuration that allows having an IDP just to identify your own pod that does not allows registration for new pods. (Would be quite easy actually, just have to remove the registration handler from the list of handlers).

If we want the user to immediately have a pod with valid WebID in the root container when they boot the server, we would need a new Initializer for this. Users could then make a custom config that includes this initializer if they want to set up a server with a pod in the root container.

Reply:
Thanks, added issues for this in https://github.com/solid/community-server/issues/828 in https://github.com/solid/community-server/issues/829
Not needed for v1.0.0 though.